### PR TITLE
Ios fix

### PR DESCRIPTION
**Pull request by BatmanNelson**
A fix for iOS, including changing the name from BlossomsCam to Blossom Cam.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,9 @@ DerivedData
 *.xcuserstate
 ios/.xcode.env.local
 
+# This is temporary until we have an app logo
+ios/BlossomsCam/Images.xcassets
+
```
Comment (BrandonDuncan13) [inline .gitignore:27]: This is a pretty cool thing to think about. Having a temporary image in the meantime is a good idea.

File: package.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     "@gorhom/bottom-sheet": "^4",
     "@react-navigation/native": "^6.0.10",
     "@react-navigation/native-stack": "^6.6.2",
-    "react": "17.0.2",
-    "react-native": "0.68.2",
+    "react": "18.0.0",
+    "react-native": "0.69.0",
```
Comment (BrandonDuncan13) [inline package.json:17]: This looks good!

File: ios/BlossomsCam/Info.plist [hunk 3]
```diff
@@ -73,5 +73,9 @@
 		<string>Octicons.ttf</string>
 		<string>Zocial.ttf</string>
 	</array>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+	</array>
```
Comment (BrandonDuncan13) [inline ios/BlossomsCam/Info.plist:79]: I still need to do device orientation on Android. It looks like you're just supporting Portrait here right? I'm fine with anything, I'll just match it for Android.

File: app.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
   "name": "BlossomsCam",
-  "displayName": "BlossomsCam"
+  "displayName": "Blossom Cam"
```
Comment (BrandonDuncan13) [inline app.json:3]: Ah, nice!

File: ios/BlossomsCam/Info.plist (hunk 3 above)
Comment (BatmanNelson) [reply ios/BlossomsCam/Info.plist:79]: Yes, I am planning on changing that but I think getting OpenCV working is a higher priority.

**Pull request merged**